h2o-docs/src/product/data-science/glm.rst

```diff
@@ -717,7 +722,7 @@ As :math:`p` closes in on 1, the Tweedie density function becomes multimodal. Th
 As a conservative condition, to ensure that the density is unimodal for most values of :math:`y,\phi`, we should have :math:`p>1.2`.
 
 Tweedie Dispersion Example
-''''''''''''''''''''''''''
+^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This might be gamma dispersion example (see the `family` in the code examples).

Reply:
Good catch -- tested out and updated this example for tweedie